storm-dist/binary/src/main/assembly/binary.xml

```diff
@@ -340,10 +340,10 @@
         </fileSet>
         <!-- $STORM_HOME/extlib -->
         <fileSet>
-            <directory>${project.basedir}/../../external/storm-kafka-monitor/target</directory>
+            <directory></directory>
```

Comment:
This change is not clear. Directory change from extlib to toollib is happening in other section. Shouldn't this be removed entirely